com.unity.ml-agents/CHANGELOG.md

```diff
@@ -13,9 +13,15 @@ and this project adheres to
 
 ### Minor Changes
 #### com.unity.ml-agents (C#)
+- The `IActuator` interface and `ActuatorComponent` abstract class were added.
+These are analogous to `ISensor` and `SensorComponent`, but for applying actions
+for an Agent. They allow you to control the action space of an Agent more
+programmatically than defining the actions in the `BrainParameters`. See
```

Comment:
Is this the old name of `Behavior Parameters`? If so can you update it to match?

Reply:
The action space sizes are technically part of the BrainParameters class
https://github.com/Unity-Technologies/ml-agents/blob/4845314ade08d9456570cc520869c04c5df78a51/com.unity.ml-agents/Runtime/Policies/BrainParameters.cs#L64-L76
although they appear in the "Behavior Parameters" inspector window. I'll reword to be more generic.

Reply:
Thanks